### ISSUE TITLE
Vanilla server downloading wrong version

### ISSUE THREAD

**Issue by Bassetz** [itzg/docker-minecraft-server#318]
To begin with I'm pretty new to all this docker thingy. I had a FTB server once but any now I'm about to get a Vanilla server. And since 1.14 has serious performance issues and so on I decided to get a 1.13.2 server. But every time I try to create a new container it downloads 1.14. I hope you understand my issue. 

Here's my line: 
docker run -d -v c:/MCVanilla:/data -e version=1.13.2 -p 25565:25565 -e EULA=TRUE -e OPS=Bassetz -e INIT_MEMORY=3G -e MAX_MEMORY=6G -e 'MOTD=Stenbackens MC Server' --name MCVanilla itzg/minecraft-server

And it spits out this in the log:
Downloading minecraft_server.1.14.jar ...

I have downloaded a 1.13.2 manually also but I can't get it so start from that any suggestions there instead?

**Comment by itzg** [comment]
The variable names are case sensitive so try that with `VERSION=1.13.2` instead of `version=1.13.2`. I'm pretty sure you'll find that fixes it since I have been using that verison lately to avoid the performance issues of 1.14.

**Comment by Bassetz** [comment]
Ye that was it, thank you so much. And btw is it normal for the 1.14 version to crash during startup especially when changing the max ram at the world loading or right before?

**Comment by itzg** [comment]
Glad to hear about the version aspect.

That's not normal, but not surprising given how much more performance intensive 1.14 seems to be. Out of curiosity, can you post some logs around the crash?

**Comment by Bassetz** [comment]
Here's the code

`PS C:\Users\Sebastian> docker run -d -v c:/MCVanilla:/data -e VERSION=1.14.1 -p 25565:25565 -e EULA=TRUE -e OPS=Bassetz -e INIT_MEMOR
Y=3G -e MAX_MEMORY=6G --name MCVanilla itzg/minecraft-server
66b7c85fd0b053ccfce46e7391caa50445192d1e2bff124374d98b96a60cdb0b
PS C:\Users\Sebastian> docker logs -f MCVanilla
Running as '1000:1000' with /data as 'drwxrwxrwx    2 0        0             4096 May 10 08:21 /data'
Checking version information.
Checking type information.
Downloading minecraft_server.1.14.1.jar ...
server.properties already created, skipping
Skipping JSON check. No files present.
Setting initial memory to 3G and max to 6G
[17:09:45] [main/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[17:09:45] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[17:09:45] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[17:09:45] [main/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[17:09:45] [main/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[17:09:45] [Server thread/INFO]: Starting minecraft server version 1.14.1
[17:09:45] [Server thread/INFO]: Loading properties
[17:09:45] [Server thread/INFO]: Default game type: SURVIVAL
[17:09:45] [Server thread/INFO]: Generating keypair
[17:09:45] [Server thread/INFO]: Starting Minecraft server on *:25565
[17:09:46] [Server thread/INFO]: Using epoll channel type
[17:09:46] [Server thread/INFO]: Preparing level "world"
[17:09:46] [Server thread/INFO]: Reloading ResourceManager: Default
[17:09:47] [Server thread/INFO]: Loaded 6 recipes
[17:09:49] [Server thread/INFO]: Loaded 811 advancements
2019/05/13 17:10:03 Done
PS C:\Users\Sebastian>`

When it comes to the 'Loaded 811 advancements' it suddenly takes quite a while to load the rest and then crashes often before prepping spawn area. When i run the command without declaring the RAM it survives the initial start but often crash right after. And always when i try to stop the container from powershell (yes, this tests is running on win10) it never saves the world. It saves things like OPS tho. And I haven't had this problem with 1.13.2 only 1.14 and 1.14.1

**Comment by itzg** [comment]
I do most of my Docker development on Win10, so that's all good for me :) 

Hmm, now that 1.14.1 is out I was going to re-try that. On my Linux server even 1.14 has been fine, but I have 8 GB RAM available on that.

**Comment by Bassetz** [comment]
Ye, I run the main server on a virtual CentOS machine but most of my testing at my desktop. I have tried with different configurations from max RAM of 2GB all the way up to 12GB and it's quite random when it works. The main thing that works most of the time is skipping the initial ram config but it still crashes when I'm trying to stop the container. Also I did a brief test today on 1.14.1 and it was exactly the same.
